### PR TITLE
Add transform-origin CSS property

### PR DESCRIPTION
**Pull request by platy11**
Should fix this:

### CODE REVIEW

File: grammars/css.cson [hunk 1]
```diff
@@ -454,7 +454,7 @@
...
...gn|box|bottom-width|bottom-style|bottom-right-radius|bottom-left-radius|bottom-color|bottom|border-width|border-top-width|border-top-style|border-top-right-radius|border-top-left-radius|border-top-color|border-top|border-style|border-spacing|border-right-width|border-right-style|border-right-color|border-right|border-radius|border-length|border-left-width|border-left-style|border-left-color|border-left|border-image|border-color|border-collapse|border-bottom-width|border-bottom-style|border-bottom-right-radius|border-bottom-left-radius|border-bottom-color|border-bottom|border|bookmark-target|bookmark-level|bookmark-label|bookmark|binding|bidi|before|baseline-shift|baseline|balance|background-size|background-repeat|background-position|background-origin|background-image|background-color|background-clip|background-break|background-attachment|background|azimuth|attachment|appearance|animation-timing-function|animation-play-state|animation-name|animation-iteration-count|animation-duration|animation-direction|animation-delay|animation-fill-mode|animation|alignment-baseline|alignment-adjust|alignment|align-self|align-last|align-items|align-content|align|after|adjust|will-change)(?![-a-z])'
```
Comment (platy11) [inline grammars/css.cson]: I'm not sure if this is the right place, but it's the best place I could find

**Comment by 50Wliu** [comment]
`transform(-(style|origin))?` should work.

**Comment by platy11** [comment]
@50Wliu Fixed that, thanks!

**Comment by 50Wliu** [comment]
Now that you added that you can remove the other `transition-origin` one.

**Comment by platy11** [comment]
@50Wliu Sorry, didn't notice that

**Comment by 50Wliu** [comment]
:rocket: Thanks!

**Pull request merged**